음양-더하기-for문.js

```diff
@@ -0,0 +1,11 @@
+function solution(absolutes, signs) {
+    let result = 0;
+        for(let i=0; i<absolutes.length; i++) {
+            if(signs[i] === true) {
+                result += absolutes[i]
+            } else {
+                result -= absolutes[i]
+            }
```

Comment:
이렇게 삼항으로 바꿔주는 것도 좋을 것 같아요
```suggestion
            signs[i] ?
                result += absolutes[i]
            :
                result -= absolutes[i]
```